src/read_utils.rs

```diff
@@ -268,14 +267,12 @@ where
                 .flexible(true)
                 .trim(csv::Trim::All)
                 .from_reader(reader);
-            let mut objects: Vec<O> = vec![];
-            for object in rdr.deserialize() {
-                let object: O = skip_error_and_log!(
-                    object.with_context(|_| format!("Error reading {:?}", path)),
-                    LogLevel::Warn
-                );
-                objects.push(object);
-            }
+            use skip_error::SkipError;
```

Comment:
Minor: Is that more idiomatic than on top? (not consistent with gtfs/read.rs)

Reply:
For some of the modifications I've made in this PR, I too wondered and might be OK to keep it as it was.

In this specific case, I'd say we avoid a `mut` variable and the type resolution seems better because we don't need to specify `O` anymore.

Reply:
However, the `use` statement in the middle might not be that idiomatic! Fixed in 024e98f43b6418ef781640ae4f0c883834c87d5c.